### ISSUE TITLE
I released a fixed new version

### ISSUE THREAD

**Issue by Xuemantou** [R3nzTheCodeGOD/R3nzSkin#1161]
Thanks by  #2 
You can close the issues you open

**Comment by qinyuntan** [comment]
Thank you very very
I test it，and it work well

**Comment by asdfbs** [comment]
> Thank you very very I test it，and it work well
how to use?

**Comment by Xuemantou** [comment]
> > Thank you very very I test it，and it work well
> > how to use?

I had introduced it in my fork repo release introduction

**Comment by in567k** [comment]
@Xuemantou i downloaded the dll and put it where u said then what ? opening game dont make it work idk
thanks in advance

**Comment by Pierreluc23** [comment]
> @Xuemantou i downloaded the dll and put it where u said then what ? opening game dont make it work idk thanks in advance

I don't know how to fix the injector so I only upload the fixed DLL file
To use it , you should put the DLL file into "/League Of Legends/Game" folder , and rename it to "hid.dll"

**Comment by in567k** [comment]
that's exactly what i said i did Pierre and it didnt self inject or work so im asking what to do :)

**Comment by Pierreluc23** [comment]
> that's exactly what i said i did Pierre and it didnt self inject or work so im asking what to do :)

You play on china server ?

**Comment by in567k** [comment]
no he didn't mention its only china server so ig it's for every server ?

**Comment by Pierreluc23** [comment]
> no he didn't mention its only china server so ig it's for every server ?

Only Tencent server 
![IMG_0340](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/168309875/a7b075fb-ce13-4ba1-99c4-7e3cfaf6bf22)

**Comment by in567k** [comment]
You just bring the best news Pierre lmao

**Comment by JDangy** [comment]
This offset will cause the game to be blocked

**Comment by Pierreluc23** [comment]
> This offset will cause the game to be blocked

Blocked ?

**Comment by Xuemantou** [comment]
> > This offset will cause the game to be blocked
> 
> Any screenshot of that or the game just freeze ?

yes actually my account was blocked for a day on Chinese server , maybe it would be detected

**Comment by WkField** [comment]
> > > This offset will cause the game to be blocked
> > 
> > 
> > Any screenshot of that or the game just freeze ?
> 
> yes actually my account was blocked for a day on Chinese server , maybe it would be detected

I just tried, and it worked well, maybe its because Nvidia 
![2b85d625c9de05f17785763906916de0](https://github.com/R3nzTheCodeGOD/R3nzSkin/assets/148404549/949a6a9f-7d8f-4380-b96a-2cbe386eb8ef)

**Comment by Xuemantou** [comment]
Buddys ! I am pleased to tell you that I changed to riot server release build , now it will work normally in any riot server **theoreticly**.
And seems like the new build will not be suspended . I have try it in China server for several games . **But I can't ensure you will not be suspended !**

**Comment by JaxonNing** [comment]
牛逼，你是我的神，新的英雄已经出现了！，hero !

**Comment by ilyablaut** [comment]
Can u tell how to use it? Thank u

**Comment by Skylength** [comment]
Tested on jp server, didn't work.I have moved the dll

**Comment by Amricomj** [comment]
all right

**Comment by Xuemantou** [comment]
> Tested on jp server, didn't work.I have moved the dll

well , I have no idea about how to use it in riot server , can you please tell me how you use it until 14.10 ?

**Comment by gwz123gu** [comment]
Can it be made into a file or dll file, and then uploaded to the network disk for us to download?

**Comment by Xuemantou** [comment]
> Can it be made into a file or dll file, and then uploaded to the network disk for us to download?

 you are in China mainland so you can't download in github ?

**Comment by gwz123gu** [comment]
> > 能不能将其制作成文件或者dll文件，然后上传到网盘供我们下载呢？
> 
> 你在中国大陆，所以不能在github下载吗？

It can be downloaded in mainland China, but I only saw the offset you sent on github and not the dll file.

**Comment by Xuemantou** [comment]
> > > 能不能将其制作成文件或者dll文件，然后上传到网盘供我们下载呢？
> > 
> > 
> > 你在中国大陆，所以不能在github下载吗？
> 
> It can be downloaded in mainland China, but I only saw the offset you sent on github and not the dll file.

I upload the dll file at my fork repo , please check in it.

**Comment by yeliheng-010** [comment]
will it be banned？

**Comment by Xuemantou** [comment]
> will it be banned？

I just tested for half an day , it turns out my account was not banned

**Comment by yeliheng-010** [comment]
> > will it be banned？
> 
> I just tested for half an day , it turns out my account was not banned

you are a real hero！！！！！！！

**Comment by Xuemantou** [comment]
Buddys , sorry to tell you that riot server can't use this skin changer **because the vanguard is introduced to every riot server** . The new version I build can be used **only in China server**.

**Comment by meteorsthink** [comment]
hero！！

**Comment by 1980862933** [comment]
in chinese servers it result ban 3- 7day

**Comment by meteorsthink** [comment]
> in chinese servers it result ban 3- 7day

Have you tested it yet? I only tested it in the training camp yesterday and there was no problem

**Comment by 1980862933** [comment]
use in rank ban 7days ,This offset will be prohibited in Chinese servers.
